docs/reference/commandline/run.md

```diff
@@ -330,6 +330,13 @@ This adds the `busybox` container to the `mynet` network.
 $ docker run -itd --net=my-multihost-network busybox
 ```
 
+You can also choose the IP addresses for the container with `--ip` and `--ip6`
+flags when you start the container on a user-defined network.
+
+```bash
+$ docker run -itd --net=my-multihost-network --ip=10.10.9.75 busybox
```

Comment:
@aboch I think using the `nginx` (or an image that *stays up* and will not need `-it`, just `-d`) image would make more sense.